test/cluster/test_tablet_stats.py

```diff
@@ -0,0 +1,83 @@
+#
+# Copyright (C) 2025-present ScyllaDB
+#
+# SPDX-License-Identifier: LicenseRef-ScyllaDB-Source-Available-1.0
+#
+from test.cluster.conftest import skip_mode
+from test.pylib.manager_client import ManagerClient
+from test.cluster.util import get_topology_coordinator, trigger_stepdown
+
+import pytest
+import logging
+
+from test.pylib.rest_client import read_barrier
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.mark.asyncio
+@skip_mode('release', 'error injections are not supported in release mode')
+async def test_load_stats_on_coordinator_failover(manager: ManagerClient):
+    cfg = {
+        'data_file_capacity': 7000000,
+        'error_injections_at_startup': ['short_tablet_stats_refresh_interval'],
+    }
+    servers = await manager.servers_add(3, config=cfg)
+    host_ids = [await manager.get_host_id(s.server_id) for s in servers]
+    cql = manager.get_cql()
+
+    coord = await get_topology_coordinator(manager)
+    coord_idx = host_ids.index(coord)
+    await trigger_stepdown(manager, servers[coord_idx])
+
+    async def get_capacity():
+        rows = cql.execute(f"SELECT * FROM system.load_per_node WHERE node = {host_ids[coord_idx]}")
+        return rows.one().storage_capacity
```

Comment:
@bhalevy @carlosms I think cloud could use this so that it gets a view on storage utilization for scale out/in which is consistent with load-balancer's view.

In the future, scylla may internally reserve some space for other things, and we better not duplicate this logic in many places.